### ISSUE TITLE
Creating a route to the given address with a default navigation app

### ISSUE THREAD

**Issue by antorix** [kivy/kivy#8090]
Hi, I would really appreciate a function to call the navigation intent that will open the default navigation app and create a route to the given address (as a string). Preferably if the user will have the ability to choose between apps, if there are more than one.

**Comment by misl6** [comment]
Duplicated of https://github.com/kivy/plyer/issues/731

**Comment by rshah713** [comment]
I've authored kivy/plyer#732 which should be what you're looking for

**Comment by antorix** [comment]
> I've authored [kivy/plyer#732](https://github.com/kivy/plyer/pull/732) which should be what you're looking for

yeah I saw it, but I can't reproduce it at the moment, my app crashes when I do `maps.route`. I will look into it later

**Comment by rshah713** [comment]
@antorix Please note the PR is only the facade. The implementation for the various platforms will be opened in a separate PR once kivy/plyer#732 is merged. 

If you are getting `NotImplementedError()` this is the correct behavior atm. What platform are you looking for?

**Comment by rshah713** [comment]
Ok, I will look into writing an implementation once the above PR is merged (or you can as well).
